docs/openapi/components/schemas/credentials/PowerOfAttorneyCredential.yml

```diff
@@ -0,0 +1,315 @@
+$linkedData:
```

Comment:
Can you add the Linked Data definitions for this credential aswell?

Reply:
See line 3...  :)